services/service-helpers/validator.js

```diff
@@ -23,6 +23,73 @@ v.validate = function(schema, config) {
   });
 };
 
+v.oneOf = function(options) {
+  if (!Array.isArray(options)) {
+    throw new Error('oneOf expects an array');
+  }
+  var checks = [];
+  options.forEach(function(option) {
+    if (!option || !option.__check) {
+      throw new Error(option + ' is not a valid option');
+    }
+    checks.push(option.__check);
+  });
```

Comment:
I'm not sure we should add code in here to validate the validation, since this doesn't help the end user, just the developer. With the ideal of making this validator as small as possible, could we could some lines and expect the developer to follow precedent and debug? (We could also document the validators.)